test/data/test_multi_nested_tensor.py

```diff
@@ -81,6 +81,30 @@ def test_multi_nested_tensor_basics(device):
     assert dense_multi_nested_tensor.shape == (multi_nested_tensor.shape[:-1] +
                                                (max_len, ))
 
+    multi_nested_tensor_with_nan = multi_nested_tensor.clone()
+    multi_nested_tensor_with_nan.values[0::2] = -1
```

Comment:
I am not sure what this line is doing. Can you clarify?

Reply:
Adding nan values to the tensor to test `fillna_col`.

Reply:
Yes I understand that, I just did not understand why `0::2`, but maybe that's just arbitrary.

Reply:
Yeah it's just arbitrary